src/main/java/io/wispforest/owo/ui/container/Containers.java

```diff
@@ -48,6 +51,21 @@ public static <C extends Component> ScrollContainer<C> horizontalScroll(Sizing h
     // Utility wrappers
     // ----------------
 
+    public static <T, C extends Component> FlowLayout list(Iterable<T> data, Consumer<FlowLayout> layoutConfigurator, Function<T, C> componentMaker, boolean vertical) {
+        var layout = vertical ? Containers.verticalFlow(Sizing.content(), Sizing.content()) : Containers.horizontalFlow(Sizing.content(), Sizing.content());
+        layoutConfigurator.accept(layout);
+
+        for (var value : data) {
+            layout.child(componentMaker.apply(value));
+        }
+
+        return layout;
+    }
+
+    public static <T, C extends Component> FlowLayout list(Iterable<T> data, Consumer<FlowLayout> layoutConfigurator, Function<T, C> componentMaker) {
```

Comment:
If we're gonna keep this method in, it at least needs some short javadoc to explain what it actually means for the resulting list's axis